src/foam/u2/tag/Button.js

```diff
@@ -30,7 +30,7 @@ foam.CLASS({
     {
       class: 'foam.u2.ColorToken',
       name: 'buttonPrimaryColor',
-      value: '$primary400',
+      value: '$primary500',
```

Comment:
This shouldnt happen here as it will impact all foam apps, make this change in your cssTokenOverrides.jrl file for your theme, or refine button.js in your repo